sycl/doc/extensions/experimental/sycl_ext_oneapi_in_order_queue_events.asciidoc

```diff
@@ -145,6 +151,9 @@ submission following a call to this member function, `externalEvent.wait()` is
 called and `externalEvent` will not be a dependency on the next command
 submitted to the queue.
 
+`externalEvent` is required to reach completed state after the event returned by
+`get_last_event()` reaches completed state.
+
```

Comment:
I assume `get_last_event` should be `ext_oneapi_get_last_event`?

I find this sentence pretty confusing.  The text you added above to `ext_oneapi_get_last_event` says:

> If the last event was set by a call to `ext_oneapi_set_external_event()`, the returned event is guaranteed to compare equal to the event argument of the  `ext_oneapi_set_external_event()` call.

Therefore, `externalEvent` is the same event as "the event returned by `get_last_event()`".

Does this text capture the requirement:

> The application is required to ensure that `externalEvent` does not reach the completed state before the completion of the most recent command that was submitted to the queue.  If this requirement is violated, the behavior is undefined.

Reply:
I agree, your suggested wording is much clearer.